### ISSUE TITLE
[install centOS] ./install.sh

### ISSUE THREAD

**Issue by maltheism** [aces/Loris#5438]
I'm running the ./install.sh script and getting this output:
> LORIS Installation Script starting at 2019-10-31T13:14:28
> The log for this session will be stored in file /var/www/loris/tools/logs/install-2019-10-31T13:14:28.log
> ---------------------------------------------------------------------
>                    LORIS Installation Script
> ---------------------------------------------------------------------
> 
> PHP does not appear to be installed. Aborting.
> 

I think it's related to this issue: https://github.com/aces/Loris/issues/5415

**Comment by maltheism** [comment]
Solution seems to be `sudo ln -s /usr/bin/php72 /usr/bin/php` to get passed this problem. I'm guessing the install.sh script could be modified to do php72 instead as another solution. @johnsaigle thoughts?

**Comment by johnsaigle** [comment]
No thoughts -- I don't have any experience with CentOS.

**Comment by driusan** [comment]
The PR sent for this was closed, I'm taking off the tag..

**Comment by johnsaigle** [comment]
@driusan I think #5492 takes over for the closed PR about so I'm going to add it back

**Comment by driusan** [comment]
Are you sure? #5492 says it's for #5383, not #5438

**Comment by johnsaigle** [comment]
True, but #5489 has a [comment](https://github.com/aces/Loris/pull/5489#issuecomment-548874342) closed in favour of #5492.

it's a bit of a maze

**Comment by maltheism** [comment]
@johnsaigle @driusan yah sorry for it being a maze. The installation process of CentOS had a couple errors and were solved from installing php from a different repo as found here https://github.com/aces/Loris/pull/5492/files. This PR should be closed as I went through the installation process a few times after that solution and the install.sh won't complain because it's finding php when installed from a different repo than originally.
